packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -22,7 +22,9 @@ import 'theme.dart';
 // RangeValues _dollarsRange = RangeValues(50, 100);
 // void setState(VoidCallback fn) { }
 
-/// A void method that paints the value indicator.
+/// [RangeSlider] uses this callback to paint the value indicator on the overlay
+/// Since the value indicator is painted on the Overlay; this method is used to
```

Comment:
This method paints the value indicator in a [RenderBox] that appears in the [Overlay].

Reply:
Done